src/main/java/org/snakeyaml/engine/v2/tokens/AnchorToken.kt

```diff
@@ -11,30 +11,21 @@
  * or implied. See the License for the specific language governing permissions and limitations under
  * the License.
  */
-package org.snakeyaml.engine.v2.tokens;
+package org.snakeyaml.engine.v2.tokens
 
+import org.snakeyaml.engine.v2.common.Anchor
+import org.snakeyaml.engine.v2.exceptions.Mark
+import java.util.*
 
-import java.util.Objects;
-import java.util.Optional;
-import org.snakeyaml.engine.v2.common.Anchor;
-import org.snakeyaml.engine.v2.exceptions.Mark;
+class AnchorToken(value: Anchor, startMark: Optional<Mark>, endMark: Optional<Mark>) : Token(startMark, endMark) {
+  @JvmField
+  val value: Anchor
 
-public final class AnchorToken extends Token {
-
-  private final Anchor value;
-
-  public AnchorToken(Anchor value, Optional<Mark> startMark, Optional<Mark> endMark) {
-    super(startMark, endMark);
-    Objects.requireNonNull(value);
-    this.value = value;
+  init {
+    Objects.requireNonNull(value)
```

Comment:
I think the null check is inserted by Kotlin, but we can clean up such stuff later.

Reply:
yeah I cleaned this one up in a later commit (just don't ask me which commit...)